server/controllers/offerController.ts

```diff
@@ -1,23 +1,11 @@
-import { Request, Response } from 'express';
-// import { Document } from 'mongoose';
```

Comment:
Impressive work! Sorry bro, but should not we remove this line (line 2) just to keep the code clean and simple as it is?

Reply:
Yeah, we definitely should